### PR TITLE
[CDAP-16446] Fixed bug with parsing int values

### PR DESCRIPTION
**Pull request by MEseifan**
Small fix for a bug when parsing int values into DLP table

### CODE REVIEW

File: src/main/java/io/cdap/plugin/dlp/Utils.java [hunk 1]
```diff
@@ -241,14 +241,14 @@ public static Table getTableFromStructuredRecord(StructuredRecord record) throws
             break;
           case INT:
           case LONG:
-            valueBuilder.setIntegerValue((Long) fieldValue);
+            valueBuilder.setIntegerValue(Long.parseLong(fieldValue.toString()));
```
Comment (albertshau) [inline src/main/java/io/cdap/plugin/dlp/Utils.java]: this should just be `(long) fieldValue)`

File: src/main/java/io/cdap/plugin/dlp/Utils.java [hunk 2]
```diff
@@ -241,14 +241,14 @@ public static Table getTableFromStructuredRecord(StructuredRecord record) throws
...
             break;
           case BOOLEAN:
             valueBuilder.setBooleanValue((Boolean) fieldValue);
             break;
           case DOUBLE:
           case FLOAT:
-            valueBuilder.setFloatValue((Double) fieldValue);
+            valueBuilder.setFloatValue(Double.parseDouble(fieldValue.toString()));
```
Comment (albertshau) [inline src/main/java/io/cdap/plugin/dlp/Utils.java]: should just be `(double) fieldValue`

File: src/main/java/io/cdap/plugin/dlp/Utils.java (hunk 1 above)
Comment (albertshau) [reply src/main/java/io/cdap/plugin/dlp/Utils.java]: assuming fieldValue cannot be null here

Comment (MEseifan) [reply src/main/java/io/cdap/plugin/dlp/Utils.java]: oh...I dont know why I though it needed the non-primitive type. Done

File: src/main/java/io/cdap/plugin/dlp/Utils.java (hunk 2 above)
Comment (MEseifan) [reply src/main/java/io/cdap/plugin/dlp/Utils.java]: done

**Review by albertshau — APPROVED** [review]

**Pull request merged**